paper-icon-button.html

```diff
@@ -76,7 +76,11 @@
         position: relative;
         padding: 8px;
         outline: none;
-        -webkit-tap-highlight-color: rgba(0,0,0,0);
+
+        /* NOTE: Both values are needed, since some phones require the value to be `transparent`. */
```

Comment:
Wait... is `rgba(0, 0, 0, 0)` alone sufficient? They mentioned that the color `transparent` wasn't supported on certain versions of iOS, but is there anywhere that `rgba(0, 0, 0, 0)` isn't supported?